scripts/test/hyriseConsole_test.py

```diff
@@ -129,11 +129,11 @@ def main():
     console.sendline("select * from meta_tables")
     console.expect("0 rows total")
 
-    # Test SSB generation.
-    console.sendline("generate_ssb 0.01")
-    console.expect("Generating tables done")
+    # Test TPC-DS generation.
+    console.sendline("generate_tpcds 1")
+    console.expect("Generating tables done", timeout=300)
```

Comment:
Do you have a rough estimate how long SF 1 generation takes?

Reply:
```
real 231.16
user 1443.67
sys 264.70
```

in docker on imres